crates/sql-backend-handler/src/sql_migrations.rs

```diff
@@ -1112,6 +1113,24 @@ async fn migrate_to_v10(transaction: DatabaseTransaction) -> Result<DatabaseTran
     Ok(transaction)
 }
 
+async fn migrate_to_v11(transaction: DatabaseTransaction) -> Result<DatabaseTransaction, DbErr> {
+    let builder = transaction.get_database_backend();
+    // Add login_enabled column to users table, defaulting to true (enabled)
```

Comment:
Can you add a check that there is no custom attribute with name "login_enabled"?